src/RevTracer.cc

```diff
@@ -0,0 +1,336 @@
+//
+// _RevTracer_cc_
+//
+// Copyright (C) 2017-2023 Tactical Computing Laboratories, LLC
+// All Rights Reserved
+// contact@tactcomplabs.com
+//
+// See LICENSE in the top level directory for licensing details
+//
+//
+
+#include <sstream>
+#include <iomanip>
+#include <string>
+
+#include "../include/RevTracer.h"
+#include "RevTracer.h"
+
+using namespace SST::RevCPU;
+
+RevTracer::RevTracer(std::string Name, SST::Output *o)
+    : name(Name), pOutput(o), outputEnabled(false), lastPC(0), insn(0), 
+    traceSymbols(nullptr), startCycle(0), cycleLimit(0), traceCycles(0), 
+    disabled(0) {
```

Comment:
Also generally, using default member initializers is clearer and less error-prone than listing out all constant initializations in a constructor. (Fill in types and put this in `RevTracer.h`; then the constructor(s) don't need to initialize these members.)

```c++
    traceSymbols = nullptr;
    startCycle = 0;
    cycleLimit = 0;
    traceCycles = 0;
    disabled = 0;
```